### PR TITLE
feat(component): add a tag to Drawer subtitle

### PR DESCRIPTION
**Pull request by sgendre**
**What is the problem this PR is trying to solve?**
A tag is need for TPD
**What is the chosen solution to this problem?**
Provide a prop to drive tag label by hosting app

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Drawer/Drawer.component.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import omit from 'lodash/omit';
 import noop from 'lodash/noop';
 import { Transition } from 'react-transition-group';
 import classnames from 'classnames';
+import { TagDefault } from '@talend/design-system';
```
Comment (Benrajalu) [inline packages/components/src/Drawer/Drawer.component.js:8]:
```suggestion
import { TagDefault, StackHorizontal } from '@talend/design-system';
```

File: packages/components/src/Drawer/Drawer.component.js [hunk 2]
```diff
@@ -101,15 +102,21 @@ export function cancelActionComponent(onCancelAction, getComponent) {
...
+export function SubtitleComponent({ subtitle, subtitleTagLabel }) {
 	if (!subtitle || !subtitle.length) {
 		return null;
 	}
-	return <h2 title={subtitle}>{subtitle}</h2>;
+	return (
+		<div className={css('inline-subtitle')}>
+			<h2 title={subtitle}>{subtitle}</h2>
+			{subtitleTagLabel ? <TagDefault>{subtitleTagLabel}</TagDefault> : null}
+		</div>
```
Comment (Benrajalu) [inline packages/components/src/Drawer/Drawer.component.js:113]:
```suggestion
		<StackHorizontal gap="XXS">
			<h2 title={subtitle}>{subtitle}</h2>
			{subtitleTagLabel ? <TagDefault>{subtitleTagLabel}</TagDefault> : null}
		</StackHorizontal>
```

File: packages/components/src/Drawer/Drawer.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@use '~@talend/design-tokens/lib/_tokens.scss' as tokens;
+
```
Comment (Benrajalu) [inline packages/components/src/Drawer/Drawer.scss:2]:
```suggestion
```

File: packages/components/src/Drawer/Drawer.scss [hunk 4]
```diff
@@ -172,3 +174,7 @@ $tc-drawer-z-index: $drawer-z-index !default;
 :global(.tc-with-drawer-wrapper:last-child) :global(.tc-drawer.stacked)::after {
 	content: none;
 }
+
+.inline-subtitle {
+	display: inline-flex;
+}
```
Comment (Benrajalu) [inline packages/components/src/Drawer/Drawer.scss:180]:
```suggestion
```

**Review by Benrajalu — COMMENTED** [review]
You could save yourself some CSS with `StackHorizonal`.

**Comment by sgendre** [comment]
> You could save yourself some CSS with `StackHorizonal`.

thx, I give a try

**Review by Benrajalu — APPROVED** [review]

**Review by tlnd-mhuchet — APPROVED** [review]

**Pull request merged**